ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -202,6 +204,24 @@ const char * snpe_subplugin::runtimeToString (zdl::DlSystem::Runtime_t runtime)
   }
 }
 
+/**
+ * @brief Internal method to get names of output layers from tensors information.
+ */
+bool snpe_subplugin::set_output_layer_names (const GstTensorsInfo *info)
+{
+  for (unsigned int i = 0; i < info->num_tensors; ++i) {
```

Comment:
If layers are NOT tensors, why are you using "num_tensors" and use GstTensorsInfo's tensor names for layer names?

Reply:
The example for convert tensorflow model into dlc model:
```
snpe-tensorflow-to-dlc --input_network ssdlite_mobilenet_v2.pb -d Preprocessor/sub "1,300,300,3" --out_node "concat" --out_node "concat_1" --output_path ssdlite_mobilenet_v2.dlc --allow_unconsumed_nodes
```
In this case, two `out_node`s have LAYER names of "concat" and "concat_1" and each LAYER has "concat:0" and "concat_1:0" as its TENSOR names.
I think the user only deal with LAYER, not with TENSOR, so LAYER could be compatible with TensorInfo.

Reply:
If you have output layers "layerA" and "layerB" and layerA has two tensors and layerB has three tensors, how are you going to handle?

Reply:
If this works well with multiple multi-tensor layers, it's good to me.

Reply:
There could be a model which contains some output nodes that have multiple meaningful output tensors, but not that I know of. Let me test with some self-made models.

Reply:
@myungjoo 
After some investigation, I found that:

In every TensorFlow model I checked, all output layers have a single output tensor. This could be also checked with the famous keras model build api `Sequential.add`.
It throws `ValueError` in case the layer argument has multiple output tensors (https://www.tensorflow.org/api_docs/python/tf/keras/Sequential#add)

So I think the multi-tensor output layer feature could be supported later with some other PR.